integration/query_test.go

```diff
@@ -1063,3 +1063,30 @@ func TestQueryIDDoc(t *testing.T) {
 	actual = FilterAll(t, ctx, collection, bson.D{{"_id", bson.D{{"z", int32(4)}, {"a", int32(3)}}}})
 	AssertEqualDocumentsSlice(t, expected, actual)
 }
+
+func TestQueryTailableCursors(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+
+	_, err := collection.InsertOne(ctx, bson.D{{"v", int32(42)}})
+	require.NoError(t, err)
+
+	expectedErr := mongo.CommandError{
+		Code:    238,
```

Comment:
What is not clear to me in the initial issue is the error code that should be returned. 
Shouldn't we return the same error code as MongoDB? That tailable cursors are not supported for capped collections in principle rather than a "non-implemented" error? I think "non implemented" error is for something we might want to implement in the future.

Reply:
we don't support tailable cursors at all.

I wanted to return unimplemented only if it's ran against capped collection, the problem here is we handle unimplemented on the `FindParam` struct tags, and return them while unmarshaling.

Reply:
> Definition of Done
> the handler updated;

To return the same error when tailable cursor is requested for non-capped collection